docs/docs/queries/structure.md

```diff
@@ -1,44 +1,207 @@
 # Structure of a Query
 
-The dataview query language is a simple, structured, custom query language for quickly creating views on your data. It
+Dataview provides [multiple ways](dql-js-inline.md) to create queries and the way you need to write a query differs for each.
```

Comment:
While this is great, it feels a bit too conversational. I'm thinking of something like:

"Dataview offers multiple ways to write queries but they use different styles and conventions."

"

Reply:
"use different styles and conventions" feels overly complicated to get "you need to write each differently". I tried with "the syntax differs for each" - what do you think, is syntax a too technical term?

Reply:
Nah I think yours is better.